modules/xfem/test/tests/moving_interface/verification/tests

```diff
@@ -21,6 +21,17 @@
     design = 'LevelSetCutUserObject.md'
     issues = '#12833'
   [../]
+  [./1D_xy_discrete2mat]
+    type = Exodiff
+    input = 1D_xy_discrete2mat.i
+    exodiff = '1D_xy_discrete2mat_out.e 1D_xy_discrete2mat_out.e-s002 1D_xy_discrete2mat_out.e-s003 1D_xy_discrete2mat_out.e-s004 1D_xy_discrete2mat_out.e-s005 1D_xy_discrete2mat_out.e-s006 1D_xy_discrete2mat_out.e-s007'
```

Comment:
do you need to check exodiff for all the files?

Reply:
As far as I know, in order to make sure there is no change at each timestep, we need to compare all of the files. I believe all the other tests compare all generated exodus files, but if you have another suggestion, I'll change it to whatever you think is best.

Reply:
Is this test necessary to run 7 time steps?

Reply:
No, I can reduce the number of steps. How many do you recommend?